ocfweb/docs/docs/staff/procedures/toner-handling.md

```diff
@@ -0,0 +1,11 @@
+[[!meta title="Recycling Used Toner"]]
+
+OCF goes through toners quite fast, due to the heavy volume of printing from our users. In the past, we've let it accumulate inside the storage room which resulted in accidental toner spillage. The following document contains how to recycle the toners and hope to encourage rapid removal of the toners from OCF space.
+
+## Options to dispose the toners
+
+* We can dispose the toners by returning them to [Staples](http://rewards.staples.com/SRC/faqs.do) down at the Shattuck Ave. They give us $2 Staples rewards per toner, and there is maximum limit of 10 toners per month.
```

Comment:
What is the Staples link supposed to be? It seems to go to a weird page that isn't really specifically helpful.

Reply:
The link takes me to a page where it describes $2 rewards recycling program. They don't seem to have a specific landing page for the program.

Having said that, this might be a better link than the current one: https://www.staples.com/sbd/cre/marketing/sustainability-center/recycling-services/